diff --git a/dist/cjs/exports/StyleSheet/preprocess.js b/dist/cjs/exports/StyleSheet/preprocess.js
index 8bafe64256a2a53a2b7b9acb556fff61fa50bc85..9d352b462585600cd74e4d1d99321d8c6530f782 100644
--- a/dist/cjs/exports/StyleSheet/preprocess.js
+++ b/dist/cjs/exports/StyleSheet/preprocess.js
@@ -5,7 +5,6 @@ exports.__esModule = true;
 exports.preprocess = exports.default = exports.createTransformValue = exports.createTransformOriginValue = exports.createTextShadowValue = exports.createBoxShadowValue = exports.createBoxShadowArrayValue = void 0;
 var _normalizeColor = _interopRequireDefault(require("./compiler/normalizeColor"));
 var _normalizeValueWithProperty = _interopRequireDefault(require("./compiler/normalizeValueWithProperty"));
-var _warnOnce = require("../../modules/warnOnce");
 /**
  * Copyright (c) Nicolas Gallagher.
  *
@@ -143,7 +142,6 @@ var preprocess = exports.preprocess = function preprocess(originalStyle, options
 
   // Convert shadow styles
   if (options.shadow === true, style.shadowColor != null || style.shadowOffset != null || style.shadowOpacity != null || style.shadowRadius != null) {
-    (0, _warnOnce.warnOnce)('shadowStyles', "\"shadow*\" style props are deprecated. Use \"boxShadow\".");
     var boxShadowValue = createBoxShadowValue(style);
     if (boxShadowValue != null) {
       nextStyle.boxShadow = boxShadowValue;
@@ -152,7 +150,6 @@ var preprocess = exports.preprocess = function preprocess(originalStyle, options
 
   // Convert text shadow styles
   if (options.textShadow === true, style.textShadowColor != null || style.textShadowOffset != null || style.textShadowRadius != null) {
-    (0, _warnOnce.warnOnce)('textShadowStyles', "\"textShadow*\" style props are deprecated. Use \"textShadow\".");
     var textShadowValue = createTextShadowValue(style);
     if (textShadowValue != null && nextStyle.textShadow == null) {
       var textShadow = style.textShadow;
diff --git a/dist/exports/StyleSheet/preprocess.js b/dist/exports/StyleSheet/preprocess.js
index f79944885036035ced41efa2b019003ae786ebb9..05fde0fbf79624bb7d87a867cca12b872aefe3da 100644
--- a/dist/exports/StyleSheet/preprocess.js
+++ b/dist/exports/StyleSheet/preprocess.js
@@ -9,7 +9,6 @@
 
 import normalizeColor from './compiler/normalizeColor';
 import normalizeValueWithProperty from './compiler/normalizeValueWithProperty';
-import { warnOnce } from '../../modules/warnOnce';
 var emptyObject = {};
 
 /**
@@ -133,7 +132,6 @@ export var preprocess = function preprocess(originalStyle, options) {
 
   // Convert shadow styles
   if (options.shadow === true, style.shadowColor != null || style.shadowOffset != null || style.shadowOpacity != null || style.shadowRadius != null) {
-    warnOnce('shadowStyles', "\"shadow*\" style props are deprecated. Use \"boxShadow\".");
     var boxShadowValue = createBoxShadowValue(style);
     if (boxShadowValue != null) {
       nextStyle.boxShadow = boxShadowValue;
@@ -142,7 +140,6 @@ export var preprocess = function preprocess(originalStyle, options) {
 
   // Convert text shadow styles
   if (options.textShadow === true, style.textShadowColor != null || style.textShadowOffset != null || style.textShadowRadius != null) {
-    warnOnce('textShadowStyles', "\"textShadow*\" style props are deprecated. Use \"textShadow\".");
     var textShadowValue = createTextShadowValue(style);
     if (textShadowValue != null && nextStyle.textShadow == null) {
       var textShadow = style.textShadow;
