diff --git a/build/views/Sitemap.js b/build/views/Sitemap.js
index a61428d388e5b8054f9691e72eaaa33b4ac3bc34..98253cf6f04b24bcabb71f8ce1444e06121853e5 100644
--- a/build/views/Sitemap.js
+++ b/build/views/Sitemap.js
@@ -136,14 +136,21 @@ const styles = react_native_1.StyleSheet.create({
         paddingVertical: 16,
         borderBottomWidth: 1,
         borderColor: '#313538',
-        shadowColor: '#000',
-        shadowOffset: {
-            width: 0,
-            height: 3,
-        },
-        shadowOpacity: 0.33,
-        shadowRadius: 3,
-        elevation: 8,
+        ...react_native_1.Platform.select({
+            web: {
+                boxShadow: '0px 3px 8px rgba(0, 0, 0, 0.33)',
+            },
+            default: {
+                shadowColor: '#000',
+                shadowOffset: {
+                    width: 0,
+                    height: 3,
+                },
+                shadowOpacity: 0.33,
+                shadowRadius: 3,
+                elevation: 8,
+            },
+        }),
     },
     headerContent: {
         flexDirection: 'row',
